core/src/components/datetime/datetime.tsx

```diff
@@ -2000,44 +1999,28 @@ export class Datetime implements ComponentInterface {
       <div class="calendar-header">
         <div class="calendar-action-buttons">
           <div class="calendar-month-year">
-            <ion-item
+            <button
+              class={{
+                'calendar-month-year-toggle': true,
+                'ion-activatable': true,
+                'ion-focusable': true,
+              }}
               part="month-year-button"
-              ref={(el) => (this.monthYearToggleItemRef = el)}
-              button
-              aria-label="Show year picker"
-              detail={false}
-              lines="none"
               disabled={disabled}
-              onClick={() => {
-                this.toggleMonthAndYearView();
-                /**
-                 * TODO: FW-3547
-                 *
-                 * Currently there is not a way to set the aria-label on the inner button
-                 * on the `ion-item` and have it be reactive to changes. This is a workaround
-                 * until we either refactor `ion-item` to a button or Stencil adds a way to
-                 * have reactive props for built-in properties, such as `aria-label`.
-                 */
-                const { monthYearToggleItemRef } = this;
-                if (monthYearToggleItemRef) {
-                  const btn = monthYearToggleItemRef.shadowRoot?.querySelector('.item-native');
-                  if (btn) {
-                    const monthYearAriaLabel = this.showMonthAndYear ? 'Hide year picker' : 'Show year picker';
-                    btn.setAttribute('aria-label', monthYearAriaLabel);
-                  }
-                }
-              }}
+              aria-label={this.showMonthAndYear ? 'Hide year picker' : 'Show year picker'}
+              onClick={() => this.toggleMonthAndYearView()}
             >
-              <ion-label>
+              <span id="toggle-wrapper">
```

Comment:
We apply `display: inline-flex` here so I opted to use `span` which is for inline styles instead of a `div` since that is for block styles (doesn't really matter since we override `display`, but I felt `span` was still semantically closest).

Reply:
What's the reasoning for the use of an `id` here vs. a class? I know we've used id in the past for associating labels to controls, but rarely for styling specific concerns.

Reply:
No big reason. I just figured there was only going to be one instance of this, so an `id` made the most sense.

I can also get rid of the id altogether and do `.calendar-month-year-toggle > span` though if we end up adding another span then this may cause issues.

Reply:
I think an `id` is fine here 👍 I was just curious.